server/area_manager.py

```diff
@@ -525,22 +528,28 @@ def change_lights(self, new_lights, initiator=None):
             self.lights = new_lights
             self.change_background(intended_background, validate=False) # Allow restoring custom bg.
 
+            # for now, only zone_lights has the area command
+            # making a separate receive_notifs variable just in case other functions require
+            # notification muting, but not area broadcasting.
+            receive_notifs = (area == None)
```

Comment:
I think `receive_notifs` as a variable name is a bit ambiguous (who is affected by this receive notifications variable?). I know in this case you mean the initiator, but there's probably a better name for this.